java/src/main/java/com/powsybl/python/network/NetworkModificationCFunctions.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.python.network;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.BusbarSection;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.VoltageLevel;
+import com.powsybl.python.commons.CTypeUtil;
+import com.powsybl.python.commons.Directives;
+import com.powsybl.python.commons.PyPowsyblApiHeader;
+import org.apache.commons.lang3.Range;
+import org.graalvm.nativeimage.IsolateThread;
+import org.graalvm.nativeimage.ObjectHandle;
+import org.graalvm.nativeimage.ObjectHandles;
+import org.graalvm.nativeimage.c.CContext;
+import org.graalvm.nativeimage.c.function.CEntryPoint;
+import org.graalvm.nativeimage.c.type.CCharPointer;
+import org.graalvm.nativeimage.c.type.CIntPointer;
+
+import java.util.*;
+
+import static com.powsybl.iidm.modification.topology.TopologyModificationUtils.*;
+import static com.powsybl.python.commons.Util.createIntegerArray;
+import static com.powsybl.python.commons.Util.doCatch;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+@CContext(Directives.class)
+public final class NetworkModificationCFunctions {
+
+    private NetworkModificationCFunctions() {
+    }
+
+    @CEntryPoint(name = "getConnectablesOrderPositions")
+    public static PyPowsyblApiHeader.ArrayPointer<PyPowsyblApiHeader.SeriesPointer> getConnectablesOrderPositions(IsolateThread thread, ObjectHandle networkHandle,
+                                                     CCharPointer voltageLevelId, PyPowsyblApiHeader.ExceptionHandlerPointer exceptionHandlerPtr) {
+        return doCatch(exceptionHandlerPtr, () -> {
+            String voltageLevelIdStr = CTypeUtil.toString(voltageLevelId);
+            Network network = ObjectHandles.getGlobal().get(networkHandle);
+
+            VoltageLevel voltageLevel = network.getVoltageLevel(voltageLevelIdStr);
+            Map<String, List<Integer>> feederPositionsOrders = getFeederPositionsByConnectable(voltageLevel);
```

Comment:
there is a problem here, getFeederPositionsByConnectable return a Map<String, Integer>